add-nightly-and-push.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# Adds a nightly binary to a local checkout of solc-bin repository, updates
+# file lists, commits and pushes the changes to the remote repository.
+#
+# The script expects to find the working copy in its working directory with
+# the right branch already checked out and all the details necessary to make
+# a commit and push it (committer name, `origin` remote, etc.) configured.
+#
+# The script fails if the specified version is already present in the
+# repository.
+#------------------------------------------------------------------------------
```

Comment:
Should I add the license boilerplate here? Other scripts in this repo don't have it. And it might need to be reworded slightly (not sure).